.circleci/cleanup-fixtures.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-TERMINUS_SITE=build-tools-$CIRCLE_BUILD_NUM
+TERMINUS_SITE=build-tools-$CIRCLE_PROJECT_USERNAME-$CIRCLE_BUILD_NUM
```

Comment:
Not sure why `$CIRCLE_PROJECT_USERNAME` is introduced here. Related to your fork of this project perhaps?